test/acceptance/to-js-test.js

```diff
@@ -56,4 +56,29 @@ describe('acceptance - to-js', function() {
       }).catch(done);
     });
   });
+
+  it('handles option --merge', function(done) {
+    fs.copySync('test/fixtures/locales', 'tmp/locales', { clobber: true });
+    let ps = spawn(process.execPath, [
+      'bin/ember-i18n-csv.js',
+      'to-js',
+      '--csv-path=test/fixtures/i18n-merge.csv',
+      '--locales-path=tmp/locales',
+      '--merge'
+    ]);
+
+    let out = '';
+    let err = '';
+    ps.stdout.on('data', buffer => out += buffer);
+    ps.stderr.on('data', buffer => err += buffer);
+
+    ps.on('exit', () => {
+      expect(out).to.equal('');
+      expect(err).to.equal('');
+      areDirsEqual('tmp/locales', 'test/fixtures/locales').then(areSame => {
```

Comment:
It's because your source on line 61, "test/fixtures/locales", doesn't have any missing keys. You should probably use "test/fixtures/locales-with-missing-keys" as your source to properly test. Then your merge file should have additional keys, but also some missing keys to test that the source keys are preserved.